package.json

```diff
@@ -113,9 +139,23 @@
     "rollup-plugin-node-globals": "^1.4.0",
     "rollup-plugin-terser": "^7.0.2",
     "rollup-plugin-visualizer": "^5.6.0",
-    "size-limit": "^7.0.8"
+    "rollup-plugin-generate-html-template": "1.7.0",
+    "rollup-plugin-livereload": "2.0.5",
+    "rollup-plugin-serve": "2.0.1",
+    "size-limit": "^7.0.8",
+    "standard-version": "9.5.0"
   },
   "lint-staged": {
-    "*.js": "prettier --write"
+    "*.js": "eslint --cache --fix",
+    "*.{json,js,md,ts}": "prettier --write"
+  },
+  "config": {
+    "commitizen": {
+      "path": "./node_modules/@digitalroute/cz-conventional-changelog-for-jira",
+      "jiraMode": false,
+      "jiraOptional": true,
+      "skipScope": false,
```

Comment:
Let's keep the scope optional?

Reply:
Was thinking to have the scope as integration, core, service-worker but happy to start with optional for time being